augly/video/helpers/intensity.py

```diff
@@ -211,9 +213,7 @@ def overlay_emoji_intensity(
 
 
 def overlay_onto_background_video_intensity(
-    overlay_size: Optional[float],
-    metadata: Dict[str, Any],
-    **kwargs,
+    overlay_size: Optional[float], metadata: Dict[str, Any], **kwargs,
```

Comment:
```suggestion
    overlay_size: Optional[float], metadata: Dict[str, Any], **kwargs
```

no need for trailing commas when all params are in the same line